### DISCUSSION TITLE
Padrões de construção de API's

### DISCUSSION THREAD

**Discussion by ThiagoBussola** [TI-UNICESUMAR/2024-topicos-em-programacao-ESOFT7S#2] [Web Development]
Em sala de aula vimos um pouco sobre o que são API's e o padrão REST.

Faça uma breve pesquisa e escreva como devem ser construidas API's REST e também pesquise outros padrões de construção de API's, quais suas diferenças para o padrão REST e quais suas vantagens e cenários de uso.

**Comment by ThiagoArcoverde** [comment]
Fiz um pequeno artigo no Medium sobre o tópico, segue o link para quem tiver interesse: https://link.medium.com/WgepDP37cHb

**Comment by NatanColleoni** [comment]
Fala professor! Me inspirei ai no Thiago e aproveitei pra pesquisar um pouco mais sobre GraphQL e escrever um artigo no Medium também! 
Segue o link https://link.medium.com/rhcYUICpdHb

**Reply by ThiagoArcoverde** [reply]
Brabo demais, boa Natan
